v1/github/repository/web-commit-signoff.rego

```diff
@@ -0,0 +1,61 @@
+package verify
+
+import future.keywords.in
+
+default allow := false
+
+default violations := []
+
+default desired_value := false
+
+desired_value {
+    input.config.args.desired_value
+} 
+
+verify = v {
+	v := {
+		"allow": allow,
+		"violation": {
+			"type": "Missconfiguration",
+			"details": violations,
+		},
+		"summary": [{
+			"allow": allow,
+			"reason": reason,
+			"violations": count(violations),
+		}],
+	}
+}
+
+allow {
+	count(violations) == 0
+}
+
+reason = v {
+	allow
+	v := "All values of the binary field are properly set"
```

Comment:
Since it is a specific field, please write it's name, and maybe provide a link to where Github explains this. Otherwise it is cryptic.

Reply:
See the `description` field

Reply:
I see, I suspect the term "binary field" will cause confusion, so the options I see are either mentioning the field name itself, or to simpley reply with "Values properly set" or "Values not properly set"